platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -274,6 +263,14 @@ following which implementers should keep in mind.
 1. Platform teams must seek support of enterprise leadership and show impact on
    value streams
 
+When adopting platforms, choosing the right capabilities and experiences to
```

Comment:
Would this fit better _after_ the following paragraph? One reason is that in the current order of the preceding list of challenges this item - "choosing the right priorities" - is no. 2. If you agree and move it let's add back a transition word to the beginning of the paragraph like "Another challenge when adopting platforms is...."

Or @lianmakesthings do you feel this item should be listed first? In that case let's switch the order of the preceding list too. In my opinion the existing order makes sense - treat platforms as products and gather user feedback first as that should influence priorities too.

Reply:
Going from my gut, I would probably keep this order and then change the list

Reply:
@lianmakesthings can you describe why you prefer the new order? Thanks!

I lean to the existing order because the top problem I've seen amongst early platform builders is that they choose and build capabilities without enough input from and iteration with product and application teams.  I've been surprised that these teams haven't successfully inculcated a product mindset despite all the literature on "platform as product".

I'd lean to putting "platform as product" first then following it with "prioritize carefully", cause only once a team has determined the right potential work items - i.e., by understanding customer needs - can they prioritize them.

Reply:
Sure! This is a very good discussion :)

The reason why I wanted to put this section about capabilities before the others was because in my mind the other two parts were diving deeper into the platform team and how they should be set up. 

Now that you defined the (now) second section as "platform as a product", I'd be inclined to agree with you on the order. 
But I would tease that out more in the text itself. To me, it only talks only about how the platform team should work with stakeholders. The idea of "productisation" is imho not obvious enough to people who are not familiar with the topic.

Reply:
I hear you. WDYT of the following suggestion to clarify the "platform as product" paragraph?